text/wrap.go

```diff
@@ -111,7 +110,7 @@ func appendChar(char rune, wrapLen int, lineLen *int, inEscSeq bool, lastSeenEsc
 
 		// increment the line index if not in the middle of an escape sequence
 		if !inEscSeq {
-			*lineLen++
+			*lineLen += RuneWidth(char)
```

Comment:
@L0nm4r this line was the buggy line resulting in misaligned stuff during wrapping.